docs/apache-airflow/deferrable-operator-ref.rst

```diff
@@ -0,0 +1,355 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+Supported Deferrable Operators
+==========================================
+
+List of operators that supports deferrable mode
+
+.. START automatically generated
+
+**apache-airflow**
+
+.. list-table::
+   :header-rows: 1
+
+   * - Modules
+     - Operators
+
+   * - :mod:`airflow.operators.trigger_dagrun`
+     - :mod:`TriggerDagRunOperator`
+
+
+**apache-airflow-providers-amazon**
+
+.. list-table::
+   :header-rows: 1
+
+   * - Modules
+     - Operators
+
+   * - :mod:`airflow.providers.amazon.aws.sensors.emr`
+     - :mod:`EmrContainerSensor`
+
+   * - :mod:`airflow.providers.amazon.aws.sensors.emr`
+     - :mod:`EmrJobFlowSensor`
+
```

Comment:
This list will be almost always not up to date because provider release cycle and core is not sync.
If we want to have such list it should be part of the provider docs and linked from core similar to what we do with notifiers
https://airflow.apache.org/docs/apache-airflow-providers/core-extensions/notifications.html#notifications

Reply:
Oh. Just noticed you commented the same @eladkal :)